torchci/components/benchmark/llms/SummaryPanel.tsx

```diff
@@ -165,18 +169,22 @@ export function SummaryPanel({
 
                   // Compute the percentage
                   const target = v.r.target;
-                  const lPercent = target
-                    ? `(${Number((l * 100) / target).toFixed(0)}%)`
-                    : "";
-                  const rPercent = target
-                    ? `(${Number((r * 100) / target).toFixed(0)}%)`
-                    : "";
+                  const lPercent =
+                    target && target != 0
+                      ? `(${Number((l * 100) / target).toFixed(0)}%)`
+                      : "";
+                  const rPercent =
+                    target && target != 0
+                      ? `(${Number((r * 100) / target).toFixed(0)}%)`
+                      : "";
+                  const showTarget =
+                    target && target != 0 ? `[target = ${target}]` : "";
                   const isNewModel = l === 0 ? "(NEW!)" : "";
 
                   if (lCommit === rCommit || l === r) {
-                    return `${r} ${rPercent} [target = ${target}]`;
+                    return `${r} ${rPercent} ${showTarget}`;
                   } else {
-                    return `${l} ${lPercent} → ${r} ${rPercent} [target = ${target}] ${isNewModel} `;
+                    return `${l} ${lPercent} → ${r} ${rPercent} ${showTarget} ${isNewModel} `;
```

Comment:
what are the thresholds for highlighting a slot to green (improvement) and read (regression) on the dashboard?

Reply:
That comes from this config https://github.com/pytorch/test-infra/blob/main/torchci/components/benchmark/llms/common.tsx#L31 where the cell is highlighted when the value before and after differ more than 5%.  It's just a fixed value atm.